v3/src/components/app.tsx

```diff
@@ -46,6 +48,11 @@ registerTileTypes([])
 
 export const App = observer(function App() {
   useKeyStates()
+  // default behavior is to show the user enty modal when CODAP is loaded
+  // We close the modal if user imports, drags a document, opens a document
+  // or plugin using url params
+  const {isOpen, onOpen, onClose} = useDisclosure()
```

Comment:
```suggestion
  const {isOpen: isOpenUserEntry, onOpen: onOpenUserEntry, onClose: onCloseUserEntry} = useDisclosure()
```
Makes the code below more readable.